assets/js/base/components/checkbox-list/index.tsx

```diff
@@ -25,13 +39,13 @@ import './style.scss';
  */
 const CheckboxList = ( {
 	className,
-	onChange = () => {},
+	onChange = () => void null,
```

Comment:
Just to keep things consistent, what about `void 0`? Like discussed in this thread: https://a8c.slack.com/archives/C8X6Q7XQU/p1638213487071400

I don't mind `void null` either, but we should choose one :)